source/layouts/layout.slim

```diff
@@ -66,5 +66,4 @@ html.no-js lang="en"
         br
         = image_tag "https://cdn.dnsimple.com/assets/resolving-with-us/logo-dark.png", alt: "DNSimple", style: "width:100px;"
     = partial 'google_analytics'
```

Comment:
It was also tempting to remove this. Not sure if there's any use for analytics 🔍